dexs/dexter-tezos/index.ts

```diff
@@ -0,0 +1,36 @@
+import { FetchOptions, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import { httpGet } from "../../utils/fetchURL";
+
+let _data: any
+
+async function getData() {
+  if (!_data)
+    _data = httpGet("https://github.com/StableTechnologies/usdtz-stats/blob/main/temp/dollarizedRevenue_3.json")
```

Comment:
so, when you are running it locally, you can replace this line with merged file data